lib/midcom/helper/datamanager2/widget/tinymce.php

```diff
@@ -177,6 +177,7 @@ private function _add_initscript($mode = 'exact')
 
         // Compute the final script:
         $script = <<<EOT
+var original = "";
```

Comment:
if there is more than one instance of tinymce in the page, `original` will get overwritten